package.json

```diff
@@ -73,7 +73,12 @@
     "node": ">=8.9.3",
     "npm": ">=5.5.1"
   },
-  "config": {
-    "wp_org_slug": "woocommerce-gateway-stripe"
```

Comment:
Shouldn't we keep this at least until the `woorelease` that supports the new attribute is released?
The new version supports both, giving precedence to the new `woorelease` attribute.

Otherwise we need to make sure that everybody who releases the extension uses the development version of `woorelease`.

Reply:
Yeah, it makes sense 👍 

I assumed the whole team would be aware and they would release right after, but it can take some time.

Fixed in 45dd1d205738f934e3d05e6cdbaef38e1d95b5ba.